docs/contributor/01-10-installation.md

```diff
@@ -0,0 +1,251 @@
+
+## Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [k3d](https://k3d.io) to get a local cluster for testing, or run on a remote cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
+
+## Install kubebuilder
+
+<div tabs name="Install kubebuilder" group="kubebuilder">
+  <details open>
+  <summary label="Homebrew">
+  Homebrew
+  </summary>
+Run:
+
+   ```bash
+   brew install kubebuilder
+   ```
+  </details>
+  <details>
+  <summary label="kubectl">
+  Fetch sources
+  </summary>
+
+Run:
+
+   ```bash
+   curl -L -o kubebuilder https://go.kubebuilder.io/dl/latest/$(go env GOOS)/$(go env GOARCH)
+   chmod +x kubebuilder && mv kubebuilder /usr/local/bin/
+   ```
+  </details>
+</div>
+
+## Manual installation of Application Connector Manager
+
+1. Clone the project.
+
+   ```bash
+   git clone https://github.com/kyma-project/application-connector-manager.git && cd application-connector-manager/
+   ```
+
+2. Set the Application Connector Manager image name.
+
+   ```bash
+   export IMG=custom-application-connector-manager:0.0.1
+   export K3D_CLUSTER_NAME=application-connector-manager-demo
+   ```
+
+3. Build the project.
+
+   ```bash
+   make build
+   ```
+
+4. Build the image.
+
+   ```bash
+   make docker-build
+   ```
+
+5. Push the image to the registry.
+
+<div tabs name="Push image" group="application-connector-installation">
+  <details>
+  <summary label="k3d">
+  k3d
+  </summary>
+
+   ```bash
+   k3d image import $IMG -c $K3D_CLUSTER_NAME
+   ```
+  </details>
+  <details>
+  <summary label="Docker registry">
+  Globally available Docker registry
+  </summary>
+
+   ```bash
+   make docker-push
+   ```
+
+  </details>
+</div>
+
+6. Deploy Application Connector Manager.
+
+   ```bash
+   make deploy
+   ```
+
+## Using Application Connector Manager
+
+- Create an ApplicationConnector instance
+
+   ```bash
+   kubectl apply -f config/samples/operator_v1alpha1_applicationconnector.yaml
+   ```
+
+- Delete an ApplicationConnector instance
+
+   ```bash
+   kubectl delete -f config/samples/operator_v1alpha1_applicationconnector.yaml
+   ```
+
+- Update the ApplicationConnector properties
+
+TODO: Provide example of CR update
+
+## Build and install the Application Connector module on the local k3d cluster
+
+1. Setup local k3d cluster and local Docker registry.
+
+   ```bash
+   k3d cluster create kyma --registry-create registry.localhost:0.0.0.0:5001
+   ```
+2. Add the `etc/hosts` entry to register the local Docker registry under the `registry.localhost` name.
+
+   ```
+   127.0.0.1 registry.localhost
+   ```
+
+3. Export environment variables (ENVs) pointing to the module and the module image registries.
+
+   ```bash
+   export IMG_REGISTRY=registry.localhost:8888/unsigned/operator-images
+   export MODULE_REGISTRY=registry.localhost:8888/unsigned
+   ```
+
+4. Build the Application Connector module.
+   
+   ```bash
+   make module-build
```

Comment:
Does not work I'm getting following error. 

```
Pushing image to "registry.localhost:8888/unsigned" (158ms)
Error: could not finish component transfer: kyma-project.io/module/application-connector:v0.0.1: transferring resource 0: unable to add blob: unable to add blob: failed to push: unexpected status: 403 Forbidden
panic: runtime error: hash of unhashable type errors.ErrUnexpectedStatus
```
Maybe the port `8888` is not correct? Few lines above the port `5001` is used.